simln-lib/src/sim_node.rs

```diff
@@ -423,6 +432,15 @@ impl SimulatedChannel {
         self.get_node(forwarding_node)?
             .check_htlc_forward(cltv_delta, amount_msat, fee_msat)
     }
+
+    pub fn create_simulated_nodes(&self) -> SimulatedNodes {
+        SimulatedNodes {
+            pubkey_node_1: self.node_1.policy.pubkey,
+            node_1_info: node_info(self.node_1.policy.pubkey),
```

Comment:
I think that this can just return `(NodeInfo, NodeInfo)` because we don't need the pubkeys